pkg/identityserver/store/migrations/20241001000000_email_notification_preferences.down.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE users DROP COLUMN IF EXISTS email_notification_preferences CASCADE;
```

Comment:
Aren't all transactions now `.tx.{up,down}.sql`? cc @nicholaspcr

Reply:
Yes this is correct, all migrations now have the suffix `.tx.(up|down).sql`